src/index.js

```diff
@@ -1636,7 +1669,7 @@ const initializeFormElements = () => {
       params: [
         {
           from: accounts[0],
-          to: '0x5FbDB2315678afecb367f032d93F642f64180aa3',
+          to: `${maliciousAddress}`,
```

Comment:
all the malicious address ocurrences have been replaced by the new const variable, for clarity